requirements.yml

```diff
@@ -1,5 +1,8 @@
 ---
 roles:
-  - geerlingguy.postgresql
+  - name: geerlingguy.postgresql
+    src: https://github.com/wzzrd/ansible-role-postgresql.git
+    version: add-fedora-33-support
```

Comment:
https://github.com/pulp/pulp_installer/commit/33a260cfc65ce62e9d745e1723a74b24790cb40f#diff-0e07b1b25f764cad23d7957544a041fa24302e224358d411da645b07264f6e1b

This is how i solved the problem for nonexisting debian-testing. For molecule it probably needs to be found elsewhere, but ansible's search path is quite rich in locations.

Reply:
@mdellweg Thank you! I should have thought of that when I advised @pavelpicka . I think we should go your route @mdellweg .

Reply:
used same way for f33 for now